dumbbot.py

```diff
@@ -19,33 +21,34 @@ async def on_message(ctx):
 	print(f'{ctx.author.name} said:\"{ctx.content}\" in #{ctx.channel.name}')
 
 	'''Don't react to a message sent by the bot'''
-	if ctx.author.id != 380935311540355072:
+	if ctx.author == bot.user:
+		return
 
-		'''Paste any link sent in private server to the media channel'''
-		if ctx.content.startswith('http') and ctx.guild.id == 379321436478636034:
-			channel = bot.get_channel(380028343611031565)
-			await channel.send(ctx.content)
+	'''Paste any link sent in private server to the media channel'''
+	if ctx.content.startswith('http') and ctx.guild.id == 379321436478636034:
```

Comment:
Wouldn't be surprised if there are newer, better approaches to checking guilds now in v2. Leaving for now, but something for us to think about.